inc/dropdown.class.php

```diff
@@ -215,9 +215,9 @@ static function show($itemtype, $options = []) {
              && $params['addicon']) {
 
                $output .= "<span class='fa fa-plus-circle pointer' title=\"".__s('Add')."\"
-                            onClick=\"".Html::jsGetElementbyID('add_dropdown'.$params['name'].$params['rand']).".dialog('open');\"
+                            onClick=\"".Html::jsGetElementbyID('add_dropdown'.$field_id.$params['rand']).".dialog('open');\"
```

Comment:
`$field_id` is already prefixed by `dropdown_` and suffixed by `$params['rand']`, no need to keep them twice.
```suggestion
                            onClick=\"".Html::jsGetElementbyID('add_'.$field_id).".dialog('open');\"
```